docs/privacy/general-privacy.md

```diff
@@ -1,60 +1,139 @@
+the topic.
+
+---
+
+## Transaction Graphs
+
+The Decred blockchain contains a public ledger of all Decred transactions,
+including the amount sent in each transaction, as well as its senders and its
+recipients.
+The senders and recipients are not identified by any real-world personal
+identity, only by cryptographic public keys.
+This provides a level of *pseudonymity*, but not necessarily true *anonymity*.
+It is possible to use this public information to build a transaction graph, a
+representation of how Decred has been moved between the various network
+participants.
+
+If one of your personal transactions is somehow connected to your real world
+identity, it may be possible to build a transaction graph which profiles more of
+your personal activity.
+An observer may be able to deduce other transactions which you have participated
+in, or they may be able to deduce your total balance.
 
-It’s important to understand, that transactions are pseudonymous, but not anonymous. That means, a transaction graph can be built up, and there exists companies that are doing that work. If at some point one of your transactions is connected to your real world identity, it becomes easy to make assumptions based on that, for example that other transactions also belong to you, or that a specific unspent transaction belongs to you (that could give information about your balance). 
 One possible way your personal identity could be linked to your blockchain activity is through the use of a block explorer. The administrator of the block explorer may be able to view your IP address and which addresses or transactions you are browsing, and they may infer that you own those addresses or transactions. A possible solution for this is to use blockchain explorers exclusively over Tor.
 
+---
+
 ## Transaction Broadcasting
 
-Broadcasting a transaction can reveal that the transaction you are broadcasting belongs to you. A solution for this is to broadcast your transaction through a website, over Tor. For example, <https://explorer.dcrdata.org/decodetx>
-It's also important to note, that different wallets might use different [address types](../advanced/address-details.md). This can also give a hint about your identity, which combined with other signals can ultimately reveal your identity.
+When you broadcast a transaction from your dcrd node, it is possible that other
+peers on the network may deduce that the transaction belongs to you, and they
+may be able to link it to your public IP address.
+It is possible to route your dcrd traffic through Tor to mitigate this risk.
+Another solution is to broadcast your transaction through a public website,
+using Tor to provide anonymity.
+For example, [dcrdata](https://github.org/decred/dcrdata) has this
+functionality, and is available as a hidden service on Tor using the URL
+<http://dcrdata2opeenddl.onion>.
+
+It's also important to note, that different wallets might use different [address
```

Comment:
>...imortant to note,

Remove comma? This comma works, and I'm fine to leave in. But I think it reads better without it.